src/common/ModalDialog/styles.less

```diff
@@ -137,6 +137,16 @@
             max-width: initial;
             z-index: 0;
             padding: 0 1.5rem;
+
+            .buttons-container {
+                flex-direction: column;
+                gap: 1rem;
+            }
+        }
+
+        .action-button {
+            text-overflow: ellipsis;
```

Comment:
Needs to be on the label and also have `white-space: nowrap;`

Reply:
fixed !